### ISSUE TITLE
SMS Only DID shouldn't be allowed to have a Voice URL configured

### ISSUE THREAD

**Issue by deruelle** [Mobicents/RestComm#630]
Admin UI allows to configure Voice URL for SMS Only Enabled DIDs, this should be hidden
Similarly for Voice Only DIDs the SMS URLs should be hidden.

**Comment by hrosa** [comment]
When registering a new DID, the voice and sms URL configuration is hidden. Only USSD options are shown.

<img width="1552" alt="screen shot 2016-01-11 at 18 49 30" src="https://cloud.githubusercontent.com/assets/7796608/12244262/bb2c0cfe-b89a-11e5-80f3-1b98fb76e84c.png">

**Comment by gvagenas** [comment]
Commit is now reverted and issue is open

**Comment by deruelle** [comment]
I think this is due to #631 as well, the capabilities stored for a DID or SIP Number are not correctly stored somehow

**Comment by ghjansen** [comment]
Hey @brainslog i'm moving this one to my list ok? :) Thanks

**Comment by otsakir** [comment]
Investigated issue by creating arbitrary DIDs with various capabilities. Voice and SMS urls are displayed regardless of the actual, stored (and returned) capabilibies of the number.

@deruelle , it seems that #631 is a separate issue.

**Comment by brainslog** [comment]
@otsakir the commit for this issue was reverted, so it should show everything all the time now. 
The problem observed by @hrosa is likely related to #631 (but to locally created DIDs, not from Nexmo/other provider(s)). I'd check what capabilities are the new registered DIDs created with and if that is the problem, reapply this commit (as it has other improvements too).

**Comment by otsakir** [comment]
Indeed, @brainslog. I noticed the revert after posting the comment. Btw, from my test on staging, when a new DID is created all capabilities are false. More on that on #631 .
